src/components/MediaSettings/MediaSettings.vue

```diff
@@ -323,8 +310,12 @@ export default {
 			return !(this.hasCall && !this.isInLobby)
 		},
 
-		canRecordingTakePlace() {
-			return getCapabilities()?.spreed?.config?.call?.recording || false
+		showDeviceSelection() {
```

Comment:
Please check this computed property. It's used wrong at method `closeModal()`

Reply:
Could you please clarify this one?